src/main/java/org/jboss/aerogear/unifiedpush/utils/HttpRequestUtil.java

```diff
@@ -88,6 +126,12 @@ public static URLConnection post(String url, String encodedCredentials, String j
         conn.setRequestProperty("Authorization", "Basic " + encodedCredentials);
         conn.setRequestProperty("Content-Type", "application/json");
         conn.setRequestProperty("Accept", "application/json, text/plain");
+	    if(connectionSettings.getReadTimeout() != null) {
+		    conn.setReadTimeout(connectionSettings.getReadTimeout());
+	    }
+		if(connectionSettings.getConnectTimeout() != null) {
+			conn.setConnectTimeout(connectionSettings.getConnectTimeout());
+		}
```

Comment:
yeah i just noticed, sorry about that, i think it's because of tab/whitespace usage. looked normal in my ide when i wrote that.
i'll fix that stuff later today or tomorrow